lib/block_view_lockups.go

```diff
@@ -0,0 +1,1784 @@
+package lib
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/btcsuite/btcd/btcec"
+	"github.com/dgraph-io/badger/v3"
+	"github.com/golang/glog"
+	"github.com/holiman/uint256"
+	"github.com/pkg/errors"
+	"math"
+	"math/big"
+	"sort"
+)
+
+//
+// TYPES: LockedBalanceEntry
+//
+
+type LockedBalanceEntry struct {
+	HODLerPKID                  *PKID
+	ProfilePKID                 *PKID
+	ExpirationTimestampNanoSecs int64
+	BalanceBaseUnits            uint256.Int
+	isDeleted                   bool
+}
+
+type LockedBalanceEntryKey struct {
+	HODLerPKID                      PKID
+	ProfilePKID                     PKID
+	ExpirationTimestampUnixNanoSecs int64
```

Comment:
Sometimes you refer to it as Expiration sometimes as Unlock. I'm not sure that's needed vs sticking with Unlock for all.

Reply:
Agree this should be cleaned up. I'll move everything to "UnlockTimestamp" for consistency.